### PR TITLE
contrib: "refer this project" -> "refer to this project"

### PR DESCRIPTION
**Pull request by michaellilltokiwa**
[ci skip]

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ All types of contributions are encouraged and valued. See the [Table of Contents
 > And if you like the project, but just don't have time to contribute, that's fine. There are other easy ways to support the project and show your appreciation, which we would also be very happy about:
 > - Star the project
 > - Tweet about it
-> - Refer this project in your project's readme
+> - Refer to this project in your project's readme
```
Comment (fridis) [inline CONTRIBUTING.md:11]: Maybe contribute this to [contributing-gen](https://github.com/bttger/contributing-gen/blob/master/templates/contributing.dot )and if the reviewers there think this is good, we should change this as well.

**Review by fridis — COMMENTED** [review]
looks good, but maybe wait for feedback from contributing-gen!

File: CONTRIBUTING.md (hunk 1 above)
Comment (michaellilltokiwa) [reply CONTRIBUTING.md:11]: Maybe my english intuition is wrong here. ChatGPT writes:

> Both "refer this project" and "refer to this project" are correct phrases, but they have slightly different meanings and contexts of usage:
> 
>     "Refer this project": This implies directing someone's attention to the project. It suggests that you want someone to look at or consider the project without necessarily providing additional context or information about it.
> 
> Example: "Please refer this project to our marketing team for further review."
> 
>     "Refer to this project": This suggests using the project as a point of reference or source of information. It implies that the project contains relevant information that can be consulted or used for a specific purpose.
> 
> Example: "For more details on the budget allocation, please refer to this project document."
> 
> In many cases, "refer to this project" is more commonly used when you want someone to consult or use the project as a source of information, while "refer this project" might be used when you want someone to simply be aware of its existence or consider it for some action.

Comment (fridis) [reply CONTRIBUTING.md:11]: I am unsure myself. Maybe leave it unchanged.

**Comment by michaellilltokiwa** [comment]
closing since the original wording might be better.

**Pull request closed without merging**